problem_builder/mentoring.py

```diff
@@ -355,12 +355,27 @@ def allowed_nested_blocks(self):
         except ImportError:
             pass
 
+        message_block_shims = [
+            NestedXBlockSpec(
+                MentoringMessageBlock,
+                category='pb-message',
+                boilerplate=message_type,
+                label=get_message_label(message_type),
```

Comment:
@mtyaka Given the larger work that both options would represent, which we don't have time to address in the scope here, could we try to find a third option?

What would happen if two messages of the same type were added? Would there be an explicit validation error appearing for the bloc? If so, would it make sense to simply let the validation error inform the author that he needs to fix it - we can't prevent every editing issue proactively anyway, so this is already a pattern the author should be familiar with. And improve this case when Studio and xblock-utils support the use case?